### DISCUSSION TITLE
Start menu  (extended window)  partially cut off at the bottom of primary window

### DISCUSSION THREAD

**Discussion by emaza** [valinet/ExplorerPatcher#2665] [Q&A] [answered]
Issue Description:
When operating in an extended display setup, clicking the Windows button on extended window results in the Start menu appearing in the primary window, and it is partially cut off at the bottom.

Screenshots:
![Captura de pantalla 2024-01-08 103505](https://github.com/valinet/ExplorerPatcher/assets/3011342/b88f2dd2-d8f9-4da3-8d77-a477aded92de)

![Captura de pantalla 2024-01-08 102947](https://github.com/valinet/ExplorerPatcher/assets/3011342/5b239735-a19f-4707-8e0b-8e8d66c5309e)

Steps to Reproduce:

Open ExplorerPatcher version 22621.2861.62.2.
Set up an extended display with at least two screens.
Click on the Windows button in the extended display.

Expected Behavior:
The Start menu should appear within the extended display window without any cutoff or display issues.

Thank you for your attention to this matter. Please let me know if any additional information is required to address and resolve this issue.

Best regards,

update: 
windows version 11 22H2 (OS build 22621.2861).
explorer patcher settings: [ExplorerPatcher_22621.2861.62.2.reg.txt](https://github.com/valinet/ExplorerPatcher/files/13860592/ExplorerPatcher_22621.2861.62.2.reg.txt)
none others softwares installed that modifies the UI of windows

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.

Please include what settings you changed in explorer patcher properties application.

Please include any other software you have installed that modifies the UI of windows.

**Comment by emaza** [comment]
- windows version 11 22H2 (OS build 22621.2861).
- explorer patcher settings: [ExplorerPatcher_22621.2861.62.2.reg.txt](https://github.com/valinet/ExplorerPatcher/files/13860592/ExplorerPatcher_22621.2861.62.2.reg.txt)
- none others softwares installed that modifies the UI of windows

**Comment by pyrates999** [comment]
Please show the settings you changed in EP properties application.

**Comment by emaza** [comment]
TASKBAR: 
![image](https://github.com/valinet/ExplorerPatcher/assets/3011342/2debf9ee-e334-49ad-b826-800c5901f61f)
SYSTEM TRAY: 
![image](https://github.com/valinet/ExplorerPatcher/assets/3011342/a88010a4-daa4-4922-a27d-abaf14fe5a79)
FILE EXPLORER:
![image](https://github.com/valinet/ExplorerPatcher/assets/3011342/8a59b4dd-ba30-4e28-a48c-fe01c27b6212)
START MENU:
![image](https://github.com/valinet/ExplorerPatcher/assets/3011342/7db58324-7f07-4584-ba12-f9e795649382)
OTHER:
![image](https://github.com/valinet/ExplorerPatcher/assets/3011342/8fab0f83-0749-4ca8-b04d-5742d64fe9a5)
ABOUT:
![image](https://github.com/valinet/ExplorerPatcher/assets/3011342/ff152dc0-4995-4025-af3b-5d60622dc11f)


other settings are by default

**Comment by emaza** [comment]
![image](https://github.com/valinet/ExplorerPatcher/assets/3011342/5a5d0dfe-4580-4584-bf7c-4e26432a9b64)
![image](https://github.com/valinet/ExplorerPatcher/assets/3011342/9a0b4d0c-9111-45bc-b361-e5566a783ad7)
![image](https://github.com/valinet/ExplorerPatcher/assets/3011342/6d5856f4-f6af-4889-87c7-5d55a7e854f3)

Additional Details
The following section contains the additional details that were recorded. 

These details help accurately identify the programs and UI you used in this recording. 

This section may contain text that is internal to programs that only very advanced users or programmers may understand. 

Please review these details to ensure that they do not contain any information that you would not like others to see. 



Recording Session: ‎22/‎01/‎2024 14:21:30 - 14:21:42

Recorded Steps: 5, Missed Steps: 8, Other Errors: 0

Operating System: 22621.1.amd64fre.ni_release.220506-1250 10.0.0.0.2.48

Step 1: User left click on "Start (button)"
Program: Windows Explorer, 10.0.22621.2338 (WinBuild.160101.0800), Microsoft Corporation, EXPLORER.EXE, EXPLORER.EXE
UI Elements: Start, Start, Shell_SecondaryTrayWnd

Step 2: User left click on "Desktop (list)" in "Start"
Program: Windows Start Experience Host, 10.0.22621.2792 (WinBuild.160101.0800), Microsoft Corporation, STARTMENUEXPERIENCEHOST.EXE -SERVERNAME:APP.APPXYWBRABMSEK0GM3TKWPR5KWZBS55TKQAY.MCA, STARTMENUEXPERIENCEHOST.EXE
UI Elements: Desktop, SysListView32, SHELLDLL_DefView, Program Manager, Progman

Step 3: User left click on "Search box (edit)" in "Search"
Program: 623.27905.70.0, Microsoft Corporation, SEARCHHOST.EXE -SERVERNAME:CORTANAUI.APPXSTMWAAB17Q5S3Y22TP6APQZ7A45VWV65.MCA, SEARCHHOST.EXE
UI Elements: Search box, RichEditBox, Search, Windows.UI.Core.CoreWindow

Step 4: User keyboard input on "Search (window)" in "Search" [...]
Program: 623.27905.70.0, Microsoft Corporation, SEARCHHOST.EXE -SERVERNAME:CORTANAUI.APPXSTMWAAB17Q5S3Y22TP6APQZ7A45VWV65.MCA, SEARCHHOST.EXE
UI Elements: Search, Windows.UI.Core.CoreWindow

Step 5: User mouse wheel down on "hjklop, See work and web results, Press right to switch preview (option)" in "Search"
Program: 623.27905.70.0, Microsoft Corporation, SEARCHHOST.EXE -SERVERNAME:CORTANAUI.APPXSTMWAAB17Q5S3Y22TP6APQZ7A45VWV65.MCA, SEARCHHOST.EXE
UI Elements: hjklop, See work and web results, Press right to switch preview, Find results in search work and web, Find results in search work and web, Results, Search, Search, WebView, Search, Windows.UI.Core.CoreWindow

**Comment by pyrates999** [comment]
Do you have the taskbar set to autohide?

**Reply by emaza** [reply]
no 
![image](https://github.com/valinet/ExplorerPatcher/assets/3011342/ca967c94-0fc9-40bb-b757-d0d5172e9772)

**Comment by pyrates999** [comment]
Might be related to this: https://github.com/valinet/ExplorerPatcher/issues/1803

**Comment by emaza** [comment, marked as answer]
ok I see how to fix it changing taskbar alignment to "left"
![image](https://github.com/valinet/ExplorerPatcher/assets/3011342/1bc499d0-60bd-4d3f-82f4-9c94b2a349cb)

**Reply by pyrates999** [reply]
Please mark that as the answer then.

**Reply by emaza** [reply]
Thank you for your interest and attention regarding the issue.

**Reply by pyrates999** [reply]
You're welcome.
